src/components/App/App.js

```diff
@@ -37,18 +35,26 @@ function App() {
   }
 
   useEffect(() => {
-    getIDs()
+    getIDs();
   }, [])
 
   useEffect(() => {
-    ids && getSingleArtPiece();
+    ids.length && getSingleArtPiece(shuffleItems(ids)[0]);
```

Comment:
Opportunity for refactoring; we could consider storing the invocation of `shuffleItems` in a variable, then test one condition, and pass in the variable and appropriate index as an argument to `getSingleArtPiece`. i.e. - 
``` 
const shuffledIDs = shuffleItems(ids);

{ids.length && (
  getSingleArtPiece(shuffledIDs[0])
  getSingleArtPiece(shuffledIDs[0])
  etc...
)}
```

Reply:
Yeah, that's a good call.

Reply:
I think this is just getting something on the wall as a test and that shuffled items will live in state.  There will be one shuffle, most likely in utility file.   Agreed!